test/functional/test_framework/messages.py

```diff
@@ -428,7 +444,10 @@ def __repr__(self):
                   bytes_to_hex_str(self.scriptPubKey))
 
 
-class CTransaction():
+class CTransaction:
+    __slots__ = ("hash", "nLockTime", "nVersion", "sha256", "vin", "vout",
+                 "wit")
```

Comment:
Do we need to / should we remove 'wit'?

Reply:
resolved in different commit and brought in to this commit, fixed